django_sam/middleware.py

```diff
@@ -38,7 +40,24 @@ def static_admin_middleware(get_response):
 
         async def middleware(request):
             if files.get(request.path):
-                return FileResponse(open(files[request.path], 'rb'))
+                file_path = files[request.path]
```

Comment:
Not for now but ASGI has two relevant extensions which work a bit like X-SEND-FILE.  One allows you to hand over a fd and the server will get the os to write the data, without python getting involved (Zero Copy Send).  Even better you can return a path and have the server stream it for you (Path Send).  These are both likely to be much faster.

https://asgi.readthedocs.io/en/latest/extensions.html

Reply:
Ah, good to know, will have to have a look at some point, although not looking to complicate this library too much given it's target usage.